neo/app/controllers/Application.scala

```diff
@@ -83,10 +86,87 @@ class Application @Inject()(val wsClient: WSClient,
     }
   }
 
+  def getCity(NELat: Double, SWLat: Double, NELng: Double, SWLng: Double) = Action{
+    Ok(Application._getCity(NELat, SWLat, NELng, SWLng, cities))
+  }
+
   class Listener extends Actor {
     def receive = {
       case d: DeadLetter => println(d)
     }
   }
 
 }
+
+object Application{
+  def loadCity(file: File): List[JsValue] = {
+    val stream = new FileInputStream(file)
+    val json = Json.parse(stream)
+    val features = "features"
+    val geometry = "geometry"
+    val type_str = "type"
+    val coordinates = "coordinates"
+    val Polygon = "Polygon"
+    val MultiPolyton = "MultiPolygon"
```

Comment:
1.capitalize the first char for the static value, e.g. `features` -> `Features`.
2. move all those `val` outside function.